pasta_eln/guiStyle.py

```diff
@@ -1,5 +1,5 @@
 """ all styling of buttons and other general widgets, some defined colors... """
-from typing import Callable, Optional
+from typing import Callable, Optional, Any
```

Comment:
Organizing imports?


from typing import Callable, Optional, Any

import qtawesome as qta
from PySide6.QtCore import QByteArray, Qt  # pylint: disable=no-name-in-module
from PySide6.QtGui import QImage, QPixmap, QAction, QKeySequence, QMouseEvent  # pylint: disable=no-name-in-module
from PySide6.QtSvgWidgets import QSvgWidget  # pylint: disable=no-name-in-module
from PySide6.QtWidgets import QPushButton, QLabel, QSizePolicy, QMessageBox, QLayout, QWidget, QMenu, QVBoxLayout, \
  QHBoxLayout, QGridLayout, QFormLayout  # pylint: disable=no-name-in-module
from qt_material import get_theme

from .backend import Backend

Reply:
Will do in the final version.